src/domain/prices/prices.repository.ts

```diff
@@ -44,6 +46,18 @@ export class PricesRepository implements IPricesRepository {
 
   async getFiatCodes(): Promise<string[]> {
     const result = await this.coingeckoApi.getFiatCodes();
-    return this.fiatCodesValidator.validate(result);
+
+    const fiatCodes = this.fiatCodesValidator
+      .validate(result)
+      .map((item) => item.toUpperCase())
```

Comment:
Wonder if this should be seen as a route requirement and not a domain one (the `toUpperCase` one).

Given its usage, this is mostly a nitpick though.

Reply:
Hmm, good point, I guess that depends on whatever we consider. No strong opinion on my side, if you consider this should be moved to the routes layer we can change it of course 🙂

Reply:
Let's leave it like this for now. It's a specific use-case as is stands.